src/bootstrap/dapp-api.js

```diff
@@ -32,6 +32,15 @@ const initializeKleros = async () => {
   kleros = new Kleros(eth.currentProvider, STORE_PROVIDER, ARBITRATOR_ADDRESS)
 }
 
+const initializeBondingCurve = async () => {
+  BONDING_CURVE_ADDRESS =
```

Comment:
Why not combine this with initializeKleros? The async part of both is fetching the networkID

Reply:
Will change as suggested.